TransactionService/docker_context/apiListeners.py

```diff
@@ -20,6 +26,42 @@ async def connectRedis(app, loop):
     )
 
 
+async def initRedisHandler(app, loop):
+    app.config['redisHandler'] = RedisHandler(app.config['redisPool'], app.config['client'], config.DATABASE_SERVER_IP,
+                                              config.DATABASE_SERVER_PORT)
+
+
+async def initAudit(app, loop):
+    logger.debug('Creating audit handler')
+    app.config['audit'] = AuditHandler(
+        app.config['client'],
+        config.WEB_SERVER_NAME,
+        config.AUDIT_SERVER_IP,
+        config.AUDIT_SERVER_PORT
+    )
+
+
+async def initLegacyStock(app, loop):
+    # stockHost = "192.168.4.2"
+    stockHost = 'dummy-stock-1'
+
+    stockPort = 4444

```

Comment:
we should add the quote server port and ip to the config file

Reply:
Added and changed the reference here